src/main/java/subway/domain/Stations.java

```diff
@@ -17,4 +19,16 @@ public Optional<Station> getStationByName(String stationName) {
                 .filter(station -> station.getName().equals(stationName))
                 .findAny();
     }
+
+    @Override
+    public String toString() {
+        return "Stations{" +
+                "stations=" + stations +
+                '}';
+    }
+
+    public List<Station> getStations() {
+        return stations.stream()
+                .collect(Collectors.toList());
```

Comment:
스트림을 생성할 필요 없이 아래 코드로 대체하면 어떨까요?

```suggestion
        return new ArrayList<>(stations);
```

Reply:
아직 자바에 익숙하지 않나봅니다 🥲
좋은 의견 감사합니다. 반영해두었습니다!